src/types/onyx/Policy.ts

```diff
@@ -11,6 +11,7 @@ type Rate = {
     customUnitRateID?: string;
     errors?: OnyxCommon.Errors;
     pendingAction?: OnyxCommon.PendingAction;
+    enabled?: boolean;
```

Comment:
why can't we give these bools a default value? Is this just because we're part way through the TS migration?